docs/blocks-storage/querier.md

```diff
@@ -499,6 +499,11 @@ blocks_storage:
     # CLI flag: -blocks-storage.bucket-store.max-concurrent
     [max_concurrent: <int> | default = 100]
 
+    # Max number of inflight queries to execute against the long-term storage.
+    # THe limit is shared across all tenants. 0 to disable.
+    # CLI flag: -blocks-storage.bucket-store.max-inflight-requests
+    [max_inflight_requests: <int> | default = 0]
+
```

Comment:
this needs to be configured for queriers too?

Reply:
@yeya24 thoughts? I do agree that by definition it should be part of `store_gateway_config` as well

Reply:
I understand it is not good. We should change it at some time. No need to make changes in this pr as it might need a lot of work.
Let's create a separate issue to track and see if we should change the doc gen or move the flags.

Reply:
Created the issue https://github.com/cortexproject/cortex/issues/5559